### DISCUSSION TITLE
Fine-tuning API/UI to support Whisper

### DISCUSSION THREAD

**Discussion by programmaticsoftware** [openai/whisper#2352] [General]
Hi,
Is there any plan for the Fine-tuning API/UI to support Finetuning Whisper?

**Comment by gongouveia** [comment]
@programmaticsoftware Hello, you can use my UI [WhisperTemple](https://github.com/gongouveia/Whisper-Synthetic-ASR-Dataset-Generator) to generate new audio samples, and edit the transcriptions. Also it is created to easily train/ fine tune on the new samples.
Please check it out, support it and request new features.

**Reply by programmaticsoftware** [reply]
Hi @gongouveia 
Can I use this UI if I'm not familiar with Python?
Can I use it from a REST API?
Thank you

**Reply by gongouveia** [reply]
@gongouveia you just need to download the files, and run the UI from the console and for the training you can also use the console.

Please, if you can describe better what are your needs, I can direct you better for diferent solutions.

**Reply by programmaticsoftware** [reply]
@gongouveia 

I have a dataset of about 500 audio-text pair samples to fine-tune whisper, and I'm looking If I need to hire a Python ML expert to do the fine-tuning or I can use a REST API or a UI.

Thank you

**Reply by gongouveia** [reply]
@programmaticsoftware  there is no UI or API that I am aware of to train the Whisper. If you need someone to finentune it on your data shoot me a message, I would be glad to help.

**Reply by programmaticsoftware** [reply]
@gongouveia I've sent you an email using the email address I found on your website.
